packages/core/src/item.ts

```diff
@@ -278,6 +278,32 @@ export interface AuditResult {
     fieldIndex: number;
 }
 
+export type ItemHistoryFieldsChangedOption = "name" | "vaultId" | "fields" | "tags";
+
+export class ItemHistory extends Serializable {
+    constructor(vals: Partial<ItemHistory> = {}) {
+        super();
+        Object.assign(this, vals);
+    }
+
+    date: Date = new Date();
```

Comment:
This should be called `time`, since it contains the full date and time.

Reply:
Actually, let's call it `updated`, which should reflect the `updated` property of the item before the change.

Reply:
Aaactually, let's keep this property and rename it to `created`, as well as an additional `updated` property as described above.

Reply:
I'd rather keep it as `date` because it is a `Date` object. I'm happy to change it to `time` or `created` but not really `updated` (nor 2 date properties, because when would they be different? You're not going to change history entries)

Reply:
I think property names should be based on the information the contain, rather than the name of their type. My point is that `created` is more descriptive because it contains the information of when the history item was created. `updated` on the other hand is *supposed* to mirror the same property from the original item (although I'm open to a different name). We don't want to restore that value but I'd like to potentially use it for displaying the time line. I.e. show the time that version of the item was created rather than the time it was changed.

Again, I'd like the whole thing to be a list of snapshots, rather than a list of changes. I.e.: "What did the item look like at a certain point in time" vs "What has changed and when". Remember the main point of this feature is **not** to serve as an audit log but simply to allow people to restore a previous version of an item.